source/docs/essentials/add-a-page-client-side.md

```diff
@@ -0,0 +1,145 @@
+  <H1>Discover our recipes</H1>
+  <RecipesList />
+</div>;
+
+export default Recipes;
+```
+  <blockquote class="note">
+  Note that there is a `RecipesList` component here. It is in fact a business component that you can create by refering to <a href="#TODO">Add a business component</a>.
+  </blockquote>
+* `my-module/web/theme/pages/Recipes/index.js`: will proxy the `Recipes.js` file in order to be able to do imports on the folder directly. See [this blog post](http://bradfrost.com/blog/post/this-or-that-component-names-index-js-or-component-js/) for more context about this pattern.
+<!-- TODO add comment about code splitting and link to our documentation -->
+```jsx
+import Recipes from './Recipes.js'
+
+export default Recipes;
+```
+* `my-module/web/theme/pages/Recipes/Recipes.story.js`: will add your page to the Storybook of your application. In the case of a page it might be troublesome to setup, but it is worth it because it will improve your code quality by being in a sandboxed environment and it will help you reference the edge cases.
+```jsx
+import Recipes from "./Recipes.js";
+import { storiesOf } from "@storybook/react";
+
+storiesOf("pages.Recipes", module).add("default", () => {
+  return <Recipes />;
+});
+```
+  <blockquote class="note">
+  For a more detailed explanation of how Storybook works in the context of Front-Commerce, please refer to <a href="#TODO">Add a component to Storybook</a>.
+  </blockquote>
+
+### Map the URL to the page component
+
+Once you have created your component, you need to actually tell the application how to use it. To do so, we provide an extension point that will let you add your custom routes.
+
+To active it in your module, you need to create the file `my-module/web/moduleRoutes.js` that will export the list of routes of your module.
+
+```jsx
+import React from "react";
+import Route from "react-router/Route";
+import Recipes from "theme/pages/Recipes";
+
+export default () => [
+  <Route
+    path="/recipes"
+    exact
+    render={() => <Recipes />}
+    key="recipes"
+  />
+];
+```
+
+Once you've created your file, you can refresh your application
+(`npm run start`), and you should see your new route if you go
+to the `/recipes` URL.
+
+🎉
+
+<blockquote class="warning">
+    **Warning:** If several modules are registered in your application and several of them define the same route, the page of the first module in `.frontcommerce.js` will be displayed. This can be useful if you want to override a default feature of Front-Commerce.
+</blockquote>
+
+## Add a page using an URL with parameters
+
+Now, let's add a `RecipeDetails` page that should be mapped to the
+`/recipes/:recipe-id-or-slug`. You will need to follow the exact
+same steps as for the `/recipes` page:
+* Create a `RecipeDetails` component in the
+`theme/pages/Recipes/RecipeDetails` folder
+* Map the `/recipes/:recipe-id-or-slug` to the new component
+
+But how can you make the URL have a parameter (`:recipe-id-or-slug`)?
```

Comment:
```suggestion
**But how can you make the URL have a parameter (`:recipe-id-or-slug`)?**
```

The question is not very visible, I think it should stand out… either with bold as suggested or using a `<blockquote>`